src/management/UserTable.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import ReactTable from 'react-table';
+import 'react-table/react-table.css'
+import './UserTable.css';
+
+class UserTable extends React.Component {
+  constructor(props) {
+    super(props);
+
+    // TODO: Wire up real data.
+    this.state = { selected: {}, selectAll: 0, data: makeData() };
```

Comment:
nice! you can actually use the admin.auth().listUsers() method to retrieve the list of users for the app - there's an example of this in the setFirstUserAsAdmin method in https://github.com/GoogleCloudPlatform/barnard-language-learning-app/blob/master/functions/index.js.

Happy to help out with creating a cloud function for this if you'd like!

Reply:
[Also sent in a separate email]
Apologies for the delay! I tried drafting a cloud function to fetch users (pushed to the pull request), but it's not tested yet because seems like I don't have the right permissions. ('npm run serve_functions' fails on start with 'Authorization failed. This account is missing the following required permissions on project barnard-project'). Would you mind:
- Helping me get the permissions for this firebase project?
- Or, if you have the bandwidth, just cloning this pull request and help me debug as needed? (it's currently 12am my time; I won't get to this in the next 10-ish hours 😅).
Thanks!

Reply:
Sure, answered in email, but I'd recommend trying out the instructions in the README (https://firebase.google.com/docs/functions/local-emulator#set_up_admin_credentials_optional). If that doesn't work, let me know, and we can try and debug!